test/browser/index.js

```diff
@@ -3,7 +3,8 @@ import sinonChai from 'sinon-chai';
 
 chai.use(sinonChai);
 
-import 'babel-polyfill';
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
```

Comment:
We don't care anymore about envs that don't have native async/await. What do we need to update to get rid of this?

Reply:
removed